### ISSUE TITLE
vue-cli: ERROR RequestError: Error: getaddrinfo EAI_AGAIN registry.npmjs.org:443 RequestError: Error: getaddrinfo EAI_AGAIN registry.npmjs.org:443

### ISSUE THREAD

**Issue by xiaoyueyue165** [vuejs/vue-cli#1302]
### Version
3.0.0-beta.10

### Reproduction link
[https://codesandbox.io/s/vue](https://codesandbox.io/s/vue)

### Steps to reproduce
npm install -g @vue/cli
vue create canmou

### What is expected?
I do not know.

Forgive me, the address is copied, no reference, just failed to create the project

### What is actually happening?
````bash
npm config set registry https://registry.npm.taobao.org --global 
npm config set disturl https://npm.taobao.org/dist --global
npm config set proxy=http://proxy3.bj.petrochina:8080
````
Just changed the new computer, I have configured the agent, use Taobao image, successfully installed vue-cli, there is the following error message, hope to get help.

````bash
$ vue create canmou
 ERROR  RequestError: Error: getaddrinfo EAI_AGAIN registry.npmjs.org:443
RequestError: Error: getaddrinfo EAI_AGAIN registry.npmjs.org:443
    at new RequestError (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@vue\cli\node_modules\request-promise-core\lib\errors.js:14:15)
    at Request.plumbing.callback (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@vue\cli\node_modules\request-promise-core\lib\plumbing.js:87:29)
    at Request.RP$callback [as _callback] (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@vue\cli\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at self.callback (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@vue\cli\node_modules\request\request.js:185:22)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestError (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@vue\cli\node_modules\request\request.js:877:8)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at TLSSocket.socketErrorListener (_http_client.js:387:9)

````

**Comment by xiaoyueyue165** [comment]
The above method does not work, I use the following method to create, would also like to ask in the custom configuration how to choose the results of a variety of situations, such as I want to select Runtime-only, I tried many times did not find the key, please also Inform, thank you！



````
$ vue init webpack anm
 ......
? Author xiaoyueyue165 <1656800216@qq.com>
? Vue build (Use arrow keys)
> Runtime + Compiler: recommended for most users
  Runtime-only: about 6KB lighter min+gzip, but templates (or any Vue-specific ·
HTML) are ONLY allowed in .vue files - render functions are required elsewhere

````

**Comment by yyx990803** [comment]
Looks like your network cannot access the original npm registry. This should be fixed by 8fbbd35
